Description: Enable the gxvalid and otvalid table validation modules
 The 'ftvalid' demo in the freetype2-demos package is only useful if certain
 extra extensions are enabled in freetype. These extensions are not enabled
 upstream by default, but there's no reason for us not to enable them.
Author: Paul Wise <pabs@debian.org>
Bug-Debian: http://bugs.debian.org/520879
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/freetype/+bug/239626
Forwarded: not-needed
Last-Update: 2021-10-07

--- a/modules.cfg
+++ b/modules.cfg
@@ -117,7 +117,7 @@
 
 # TrueType GX/AAT table validation.  Needs `ftgxval.c' below.
 #
-# AUX_MODULES += gxvalid
+AUX_MODULES += gxvalid
 
 # Support for streams compressed with gzip (files with suffix .gz).
 #
@@ -136,7 +136,7 @@
 
 # OpenType table validation.  Needs `ftotval.c' below.
 #
-# AUX_MODULES += otvalid
+AUX_MODULES += otvalid
 
 # Auxiliary PostScript driver component to share common code.
 #
